MetaMask/metamask-extension
# Move security alerts out of Experimental settings on Extension

## **Description**

We should move security alerts feature out of `Settings > Experimental` to `Settings > Security & Privacy` under the Security section below the SRP reveal feature.

## **Related issues**

Fixes:  [#1764](https://github.com/MetaMask/MetaMask-planning/issues/1764)

## **Manual testing steps**

1. Go to this Settings -> Experimental
2. You should see blockaid settings
3. Go to Settings -> Security & privacy
4. You should NOT see blockaid under SRP Reveal
5. Checkout this branch
6. Go to this Settings -> Experimental
7. You should NOT see blockaid settings
8. Go to Settings -> Security & privacy
9. You should see blockaid under SRP Reveal

## **Screenshots/Recordings**

### **Before**

https://github.com/MetaMask/metamask-extension/assets/44811/ca5b399e-184b-4ba9-8d1e-4461ce13f324

### **After**

https://github.com/MetaMask/metamask-extension/assets/44811/ac8cff81-382b-492d-9f77-998f1e2bd68a

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
